enterprise/frontend/src/metabase-enterprise/whitelabel/components/ImageToggle/ImageToggle.styled.tsx

```diff
@@ -0,0 +1,27 @@
+import styled from "@emotion/styled";
+import { color } from "metabase/lib/colors";
+
+export const ToggleRoot = styled.div`
+  display: flex;
+  flex: 1 1 auto;
+  max-width: 33rem;
+  border: 1px solid ${color("border")};
+  border-radius: 0.5rem;
+`;
+
+export const ImageContainer = styled.div`
+  padding: 0.5rem 1rem 0.75rem 1.25rem;
```

Comment:
Not sure I'd justify using this padding for the lighthouse image. I guess the intention behind this unequal padding is to make the Metabot look center.
![image](https://user-images.githubusercontent.com/1937582/178915739-9bbe103c-bd30-49f3-9674-a5cff83d5a71.png)
![image](https://user-images.githubusercontent.com/1937582/178915772-47f7281f-cca4-4a3e-a2ed-77772334c81a.png)

I'd rather use more uniform padding and adjust the Metobot position separately.

Reply:
Agree, fixed